assets/js/ajax-filters.js

```diff
@@ -490,8 +486,16 @@ jQuery( document ).ready( function( $ ) {
 
 			return false;
 		} )
-		.on( 'submit', function() {
-			return false;
+		.on( 'submit', function(event) {
```

Comment:
We just changed it to stop persisting here and instead only when moving away from page on a job listing result.

Reply:
That behavior is still happening, when the filters form is on a page that contains `. job_listings`, probably in the majority of cases, the form submission should be prevented.

This PR is about cases when the filtering template is out of the [jobs]. For example when you create a widget that contains only `get_job_manager_template( 'job-filters.php' );` and use the browser's form submission process, and redirect the search from a homepage without listings to the selected Listings page. At this point, this case is impossible, because the filtering logic is bound to the listings logic.

Also, I just realized this part of code has an issue, `persist_form` should be called after the `if` on line 494, not before.